content/components/pages/RecipeDetailsPage.tsx

```diff
@@ -3,31 +3,67 @@ import { useParams } from "react-router-dom";
 
 import DetailsHeader from "devtools/components/recipes/details/DetailsHeader";
 import RecipeDetails from "devtools/components/recipes/details/RecipeDetails";
-import { useSelectedNormandyEnvironmentAPI } from "devtools/contexts/environment";
+import {
+  useSelectedNormandyEnvironmentAPI,
+  useSelectedExperimenterEnvironmentAPI,
+} from "devtools/contexts/environment";
+import { ExperimenterDetailsProvider } from "devtools/contexts/experimenterDetails";
 import { RecipeDetailsProvider } from "devtools/contexts/recipeDetails";
 
 // export default
 const RecipeDetailsPage: React.FC = () => {
   const { recipeId } = useParams<{ recipeId: string }>();
   const normandyApi = useSelectedNormandyEnvironmentAPI();
-  const [data, setData] = React.useState({});
+  const experimenterApi = useSelectedExperimenterEnvironmentAPI();
+  const [recipeData, setRecipeData] = React.useState({
+    experimenter_slug: "",
```

Comment:
The experimenter slug is not always a string, as sometimes it is `null`. I don't think that modifying the default state to include an empty string is right here.